packages/doenetml-worker-rust/lib-doenetml-core/src/core/components/common.rs

```diff
@@ -256,7 +254,7 @@ pub trait ComponentActions: ComponentNode {
     }
 }
 
-/// A `ComponentProfile` is used in the `DependencyInstruction` specifying children.
+/// A `ComponentProfile` is used in the `DataQuery` specifying children.
```

Comment:
Just starting to process this. At this point, I don't understand the difference between a "data query" and a "graph query". I assume this should be a "graph query" here, given that all the below comments use "graph query".

Reply:
I had initially named things `GraphQuery`, but then I thought `DataQuery` was a better name overall. So `GraphQuery` shouldn't show up at all.